core/src/servico/NotificacaoService.java

```diff
@@ -45,38 +45,23 @@ public String notificarAusencia(Long idProfessor, String dataInicio, String data
         SimpleDateFormat sdf = null;
         
         DateTime inicio = null;
-        DateTime inícioReal = null;
-        
         DateTime fim = null;
-        DateTime fimReal = null;
         
-        if(dataInicio.matches("(0?[1-9]|[12][0-9]|3[01])/(0?[1-9]|1[012])/((19|20)\\d\\d)"))
-        {
-            sdf = new SimpleDateFormat("dd/MM/yyyy");
-            
-            inicio = new DateTime(sdf.parse(dataInicio));
-            inícioReal = new DateTime(inicio.getYear(),inicio.getMonthOfYear(),inicio.getDayOfMonth(),00,00);
-
-            fim = new DateTime(sdf.parse(dataFim));
-            fimReal = new DateTime(fim.getYear(),fim.getMonthOfYear(),fim.getDayOfMonth(),23,59);            
-        }
-        else
-        {
+        try {
             sdf = new SimpleDateFormat("dd/MM/yyyy HH:mm");
-                        
             inicio = new DateTime(sdf.parse(dataInicio));
-            inícioReal = new DateTime(inicio.getYear(),inicio.getMonthOfYear(),inicio.getDayOfMonth(),inicio.getHourOfDay(),inicio.getMinuteOfHour());
-
             fim = new DateTime(sdf.parse(dataFim));
-            fimReal = new DateTime(fim.getYear(),fim.getMonthOfYear(),fim.getDayOfMonth(),fim.getHourOfDay(),fim.getMinuteOfHour());
+        } catch (ParseException pe) {
+            sdf = new SimpleDateFormat("dd/MM/yyyy");
+            inicio = new DateTime(sdf.parse(dataInicio)).withHourOfDay(0)
+                    .withMinuteOfHour(0);
+            fim = new DateTime(sdf.parse(dataFim)).withHourOfDay(23)

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['23' is a magic number.](https://www.codacy.com/app/calebepb/substituicao/file/2779878176/issues/source?bid=3364462&fileBranchId=3364495#l58)